docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -75,6 +75,7 @@ Ansible loads plugins automatically too, loading each type of plugin separately
 You can create or add a local plugin in any of these locations:
 
 * any directory added to the relevant ``ANSIBLE_plugin_type_PLUGINS`` environment variable (these variables, such as ``$ANSIBLE_INVENTORY_PLUGINS`` and ``$ANSIBLE_VARS_PLUGINS`` take colon-separated lists like ``$PATH``)
+* Adding or adjusting the ``library`` parameter to the ``ansible.cfg`` file
```

Comment:
I would point to the actual config setting DEFAULT_MODULE_PATH, which then details the different ways of setting it, not just the ini key w/o section information.